index.js

```diff
@@ -1,170 +1,312 @@
 #!/usr/bin/env node
-const columns = process.stdout.columns
-const rows = process.stdout.rows
+import { program } from 'commander';
+import { existsSync, writeFileSync, mkdirSync, readFileSync } from 'fs';
 
-const burshWidth = 6
-const deformationFactor = 2
-const delay = 3
-const frameSpeed = 7
+const CONFIG_DIRECTORY = `${process.env.HOME}/.wipeclean`;
```

Comment:
This breaks on windows. Id suggest using something like `require('os').homedir()` instead as the HOME value is process.env.USERPROFILE on windows.

Reply:
Also, it would be better to add this to the .config/wipeclean folder in the home directory (and an equivalent config directory in windows) instead of just spamming the home folder.

Reply:
@sekwah41 updated to use homedir. I'll save error handling for another day I think.

@TomiOhl any docs on where this .config dir should be on windows? I updated the code to create this .config dir if it doesn't exist, but can add a windows-specific solution here.

Reply:
@sekwah41 nvm added a try-catch around the writeFileSync to log if there's an error like you suggested.

Reply:
> @TomiOhl any docs on where this .config dir should be on windows? I updated the code to create this .config dir if it doesn't exist, but can add a windows-specific solution here.

App data should be under `C\Users\<Username>\AppData\Roaming\WipeClean`. See this answer too: https://softwareengineering.stackexchange.com/a/299872

Reply:
@TomiOhl thanks, I'll update to store stuff there tonight. I don't have a windows machine to test this on, I'll tag you once I make the updates and can you verify it works?

Reply:
@walkerspence sorry, me neither :D Gotta find someone else

Reply:
@TomiOhl opened this issue to handle it and disabled the -s flag on windows: https://github.com/JeanJouliaCode/wipeClean/issues/12#issue-1148790472